### PR TITLE
DEVAI-157: Pull latest updates from ai-lab-recipes

### PR DESCRIPTION
**Pull request by thepetk**
### What does this PR do?:
Simply pulls all latest updates made in `containers/ai-lab-recipes`, after the merge of https://github.com/containers/ai-lab-recipes/pull/790

### Which issue(s) this PR fixes:
DEVAI-157

### PR acceptance criteria:
Testing and documentation do not need to be complete in order for this PR to be approved. We just need to ensure tracking issues are opened and linked to this PR, if they are not in the PR scope due to various constraints.

- [ ] Tested and Verified

  

- [ ] Documentation (READMEs, Product Docs, Blogs, Education Modules, etc.)

### CODE REVIEW

File: rag/rag_app.py
Comment (Jdubrick) [inline rag/rag_app.py:79]: I know this was merged into the upstream so no point in making any changes here but just curious if we should have moved this conditional logic to a var above instead of the inline?

**Review by Jdubrick — COMMENTED** [review]
QQ before approval - Are the changes to the `Containerfiles` and `requirements.txt` changes that we hadn't previously pulled in? I know they weren't included in your upstream pr

**Review by maysunfaisal — APPROVED** [review]

File: rag/rag_app.py
Comment (thepetk) [reply rag/rag_app.py:79]: For sure it adds the same functionality, tbh I consider both options equally verbose. We could open an extra PR to update this section in case we'd like to be this way.

**Comment by thepetk** [comment]
> QQ before approval - Are the changes to the `Containerfiles` and `requirements.txt` changes that we hadn't previously pulled in? I know they weren't included in your upstream pr

Yes I think they have renovate enabled in their repo (I'd suggest doing the same with our images repo btw). See https://github.com/containers/ai-lab-recipes/commit/65a7ffe390d5da5d839a15d1aa7950a6bce0ef42 for example.

**Review by Jdubrick — APPROVED** [review]
lgtm

**Comment by thepetk** [comment]
/hold

Until proper testing is completed for DEVAI-158

**Comment by thepetk** [comment]
@maysunfaisal @Jdubrick I've updated the PR with the latest changes of the recipes https://github.com/containers/ai-lab-recipes/pull/799.

Thorough testing has been made here: https://github.com/redhat-ai-dev/ai-lab-app/pull/28

**Review by Jdubrick — APPROVED** [review]
lgtm

**Pull request merged**
